openpype/pipeline/workfile/workfile_template_builder.py

```diff
@@ -1549,24 +1559,23 @@ def populate_load_placeholder(self, placeholder, ignore_repre_ids=None):
                     repre_context["subset"],
                     repre_context["asset"],
                     loader_name,
-                    loader_args
+                    placeholder.data["loader_args"],
                 )
             )
             try:
                 container = load_with_repre_context(
                     loaders_by_name[loader_name],
                     repre_load_context,
-                    options=self.parse_loader_args(loader_args)
+                    options=loader_args
                 )
 
             except Exception:
-                failed = True
                 self.load_failed(placeholder, representation)
 
             else:
-                failed = False
                 self.load_succeed(placeholder, container)
-            self.post_placeholder_process(placeholder, failed)
+        # Run post placeholder process after load of all representations
+        self.post_placeholder_process(placeholder, failed)
 
         if failed:
```

Comment:
here `failed` doesn't necessarily exist on this scope. @tokejepsen tagging you as you were the one who made that commit, what were you expecting this to be like? if the first representation fails but the next one works, the value of `failed` gets overridden

Reply:
I have now pushed here https://github.com/ynput/OpenPype/pull/5061/commits/ac7e53a44a038df8c31ad1fbd4d49c8c90c3a95b a change so `failed` is defined on the right scope and it only gets overridden to `True` if it fails for any of the load calls

Reply:
It seems to be resolved @fabiaserra, right?